internal/commands/user/shared/inputs.go

```diff
@@ -0,0 +1,96 @@
+package shared
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/AlecAivazis/survey/v2"
+)
+
+// UsersInputs are the filtering inputs for a user command
+type UsersInputs struct {
+	State             realm.UserState
+	ProviderTypes     []string
+	Status            StatusType
+	InteractiveFilter bool
+}
+
+// ResolveUsers will use the provided Realm client to resolve the users specified by the realm.App through inputs
+func (i *UsersInputs) ResolveUsers(ui terminal.UI, client realm.Client, app realm.App) ([]string, error) {
+	var err error
+
+	userFilter := realm.UserFilter{}
+	if i.Status != StatusTypeNil {
```

Comment:
this check doesn't seem necessary since if it is true `userFilter.Pending = i.Status == StatusTypePending` will be false as well.